bip-0121.mediawiki

```diff
@@ -115,7 +115,7 @@ Send PoP for a transaction with label "video 42923" to
 <nowiki>https://www.example.com/pop/352</nowiki>, using nonce
 <tt>0x73 0xd5 0x1a 0xbb 0xd8 0x9c</tt>:
 <pre>
- btcpop:?p=https://www.example.com/pop/352&n=zgWTm8yH&label=video 42923
+ btcpop:?p=https://www.example.com/pop/352&n=zgWTm8yH&label=video%2042923
```

Comment:
I can't find anything in the specifications (BIP21 and RFC 3986) saying that space can be encoded with '+'. Please tell me where I can find that.